tests/playbooks/centos7_deps.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Install CentOS deps for network system role tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+        - git
+        - python3-pip
+        - python2-pip
+  tasks:
+    - name: Install EPEL
+      package:
+        state: present
+        name: epel-release
```

Comment:
I think this is the only difference between Fedora 32 and CentOS 8, so you can add a when: condition here and move all the installation steps into the tests/playbooks/integration_pytest_python3.yml  playbook as far as I can see.

Reply:
Also please add the installation step for EPEL with RHEL (see the EPEL wiki for the URL/command).